packages/transition-backend/src/services/osm/__tests__/OsmGetStreetsAroundPoint.test.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import { getStreetsAroundPoint } from '../OsmGetStreetsAroundPoint';
+
+jest.mock('chaire-lib-common/lib/utils/osm/OsmOverpassDownloader', () => {
```

Comment:
Here, you can return `downloadGeojson: jest.fn()`, then use a mock version of it in the tests. Search the code base for `jest.MockedFunction` to see many examples of how this is used. I think something like this would work.

`const mockedDownloadGeojson = OsmDownloader.downloadGeojson as jest.MockedFunction<typeof OsmDownloader.downloadGeojson>`

Then in each test, you can use mockedDownloadGeojson.mockResolvedValueOnce or mockedDownloadGeojson.mockRejectedValueOnce to test specific answers.

Reply:
fixed